ros2interface/ros2interface/api/__init__.py

```diff
@@ -41,6 +41,8 @@ def get_interfaces(package_name):
 
 def get_interface_path(parts):
     prefix_path = has_resource('packages', parts[0])
+    if not prefix_path:
+        raise LookupError('Unknown package {}'.format(parts[0]))
```

Comment:
Bug fixes should be done in separated PRs.

Reply:
Yes! I thought I had removed them all, looks like I didn't.

Reply:
See https://github.com/ros2/ros2cli/pull/366